src/utils/taskActions.js

```diff
@@ -0,0 +1,65 @@
+import { v4 as uuid } from "uuid";
+import { defaultNewTask } from "../components/Modal/defaultNewTask";
+
+export const validateTask = (newTask, setError) => {
+  const { taskTitle, taskDescription, taskTime } = newTask;
+
+  if (taskTitle.length === 0) {
+    setError("Title cannot be blank!");
```

Comment:
if taskTitle is your string , 
then you don't need to  use length 
you can use condition like
 if(!taskTitle){ 
setError("Title cannot be blank!")
}

Reply:
Hey thanks for the suggestion. Made the changes